Monika After Story/game/script-ch30.rpy

```diff
@@ -2004,6 +2000,8 @@ label ch30_preloop:
         set_keymaps()
 
         persistent.closed_self = False
+        # TODO: this var is always True,
```

Comment:
its set to false if the user purposefully quits the game - this is how we determine if the game crashed when `closed_self` is false. 

to check if the game just crashed, the `just_crashed` var is set a few labels above (line 1848) - or combine a not `_mas_game_crashed` with not `closed_self`

Reply:
The name is still misleading, even if user didn't crash, we get `True` in `_mas_game_crashed`. Since the purpose is actually check for force quit, should be used another var `_mas_force_quit` or similar.

Reply:
ok fair point